### PR TITLE
feat(sql): add extra sql backend

### PR DESCRIPTION
**Pull request by blancinot**
New backend ... drum roll ... SQL like database :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,3 +151,27 @@ Allowed extra claims in the etcd object:
     "email_verified": true
 }
 ```
+
+#### sql database lookup
+
+Looks up the user in the sql database.
```
Comment (mcluseau) [inline README.md]: s/sql/SQL/

File: README.md [hunk 2]
```diff
@@ -151,3 +151,27 @@ Allowed extra claims in the etcd object:
     "email_verified": true
 }
 ```
+
+#### sql database lookup
```
Comment (mcluseau) [inline README.md]: s/sql/SQL/

File: auth/sql/sql.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+type sqlAuth struct {
+	db    *sql.DB
+	table string
+}
+
+// New Authenticator with no backend
+func New(driver, dsn, table string) api.Authenticator {
+
```
Comment (mcluseau) [inline auth/sql/sql.go]: bad formatting (empty line)

**Comment by blancinot** [comment]
better :)

**Review by mcluseau — APPROVED** [review]

**Pull request merged**